spec/features/better_range_inputs_spec.rb

```diff
@@ -52,7 +52,7 @@ def slider_value
     pos = page.evaluate_script("$('#answer_v1+.noUi-target .noUi-base').position()")
     page.driver.click(pos['left'] + 100, pos['top'] + 5)
     slider_value.to_i.should be < 40
-    slider_value.to_i.should be > 10
+    slider_value.to_i.should be >= 10
```

Comment:
Maar dan klopt er iets niet, want de slider is zo'n 630px, dus op 100px moet de waarde zo'n 16+ zijn.